GUI/QtQuick/qml/Receive/ReceiveQrCode.qml

```diff
@@ -43,11 +44,20 @@ ColumnLayout  {
         Layout.alignment: Qt.AlignCenter
         Layout.topMargin: 6
         Layout.preferredHeight : 16
-        text: bsApp.walletsList[overviewWalletIndex] + " / Native SegWit"
+        text:  getWalletData(overviewWalletIndex, WalletBalance.NameRole) + " / Native SegWit"
         color: "#E2E7FF"
         font.pixelSize: 14
         font.family: "Roboto"
         font.weight: Font.Normal
+
+        Connections
+        {
+            target:walletBalances
+            function onRowCountChanged ()
+            {
+                wallet_name.text = getWalletData(overviewWalletIndex, WalletBalance.NameRole) + " / Native SegWit"
```

Comment:
Can't we avoid code duplication here and above? Use one function in both places. Better to have it as a translatable string: `qsTr("%1 / Native SegWit").arg(<wallet name>)`

Reply:
Hmm qsTr need only to Native Segwit. We will not translate wallet name I think)

Reply:
Yes, this is obvious if you look at my first comment here. Change to the suggested format please.